tests/security/ima/ima_measurement_audit.pm

```diff
@@ -42,7 +42,13 @@ sub run {
 
         # We do not check the exact file hash here, but to ensure the audit
         # record existed
-        assert_script_run("ausearch -m INTEGRITY_RULE |grep '$f->{file}.*hash='");
+        if (is_sle('>=16')) {
+            # skipping audit check on SLE 16 due to bsc#1247246
+            record_soft_failure('SKIPPING TEST; bsc#1247246');
+            return;
+        } else {
+            assert_script_run("ausearch -m INTEGRITY_RULE |grep '$f->{file}.*hash='");
```

Comment:
Just a small nitpick, but I'd add a whitespace after the `|` in `|grep` just for it to look nice.